src/mgmt/srv_drpc.c

```diff
@@ -866,7 +866,10 @@ ds_mgmt_drpc_bio_health_query(Drpc__Call *drpc_req, Drpc__Response *drpc_resp)
 	bds = bio_health->mb_dev_state;
 	resp->error_count = bds.bds_error_count;
 	resp->temperature = bds.bds_temperature;
-	resp->media_errors = bds.bds_media_errors[0];
+	if (bds.bds_media_errors)
```

Comment:
Suggestion: we probably can just use bds_media_errors[1] and use struct hack to allocate appropriate size this way we don't if condition?